.github/workflows/release-branch-tests.yml

```diff
@@ -39,8 +38,8 @@ jobs:
       max-parallel: 1
       fail-fast: false
       matrix:
-        branch: [1.0.latest, 1.1.latest, 1.2.latest, 1.3.latest, main]
-        workflow_name: [main.yml, integration.yml]
+        branch: [1.0.latest, 1.1.latest, 1.2.latest, 1.3.latest]
```

Comment:
```suggestion
        branch: [1.0.latest, 1.1.latest, 1.2.latest, 1.3.latest, 1.4.latest]
```

Reply:
Since we cut the 1.4.latest branch yesterday it needs to be added to the matrix.

Reply:
Oh good callout and yay!